types/src/errors/runtime_error.rs

```diff
@@ -133,8 +130,9 @@ enum ErrorID {
     RegularKeyAlreadyInUseAsPlatform = 20,
     ScriptHashMismatch = 21,
     ScriptNotAllowed = 22,
-    TextNotExist = 23,
-    TextVerificationFail = 24,
+    /// Deprecated
```

Comment:
I think we can remove the comments too.

Reply:
As you see right above here, there are a few errors which got the exactly same treatment. If you are ok with me only removing the comments for Text related errors, I'll do that.
```
    /// Deprecated
    // INVALID_DECOMPOSED_INPUT = 13,
    // INVALID_DECOMPOSED_OUTPUT = 14,
```
To be sure, do you mean I need to remove the `/// Deprecated` line alone or all the three lines including it altogether?

Reply:
All three lines.